custom_components/frigidaire/climate.py

```diff
@@ -290,5 +290,5 @@ def update(self):
                 self._details.for_code(
                     frigidaire.HaclCode.CONNECTIVITY_STATE
                 ).string_value
-                == frigidaire.ConnectivityState.CONNECTED
+                == frigidaire.ConnectivityState.CONNECTED.value
```

Comment:
Just thinking about this a bit more - to make this more resilient can we instead make the check

!= frigidaire.ConnectivityState.DISCONNECTED.value

This way, anything other than the DISCONNECTED value will be treated as available, including None or something invalid

Reply:
Sounds good! Done.